pontoon/base/models.py

```diff
@@ -109,16 +109,20 @@ def user_profile_url(self):
 
 def user_gravatar_url(self, size):
     email = hashlib.md5(self.email.lower().encode("utf-8")).hexdigest()
-    data = {"s": str(size)}
+    data = {"s": str(size), "d": str(404)}
 
-    if not settings.DEBUG:
-        append = "_big" if size > 88 else ""
-        data["d"] = settings.SITE_URL + static("img/anon" + append + ".jpg")
-
-    return "//www.gravatar.com/avatar/{email}?{data}".format(
+    local_gravatar_url = "https://www.gravatar.com/avatar/{email}?{data}".format(
         email=email, data=urlencode(data)
     )
 
+    r = requests.get(local_gravatar_url)
+    if r.status_code == 404:
+        return "https://eu.ui-avatars.com/api/?uppercase=true&name={name}".format(
+            name=self.display_name
+        )
+    else:
+        return local_gravatar_url
+
```

Comment:
And then you can remove this entire block.

Reply:
This is more efficient than my changes. I tried the same previously with below url:

https://eu.ui-avatars.com/api/?uppercase=false&name=different+Case

But it was not working, probably due to query string. But in your case, parameters are part of url path and am able to load the image. 

Will make the changes. Thanks.

Reply:
This is more efficient than my changes. I tried the same previously with below url:

https://eu.ui-avatars.com/api/?uppercase=false&name=different+Case

But it was not working, probably due to query string. But in your case, parameters are part of url path and am able to load the image.

Will make the changes. Thanks.